aiohue/sensors.py

```diff
@@ -53,3 +71,367 @@ def state(self):
     @property
     def config(self):
         return self.raw['config']
+
+
+class GenericCLIPSensor(GenericSensor):
+    def __init__(self, id, raw, request):
+        super().__init__(id, raw, request)
+        self._on = raw['config'].get('on')
+
+    @property
+    def on(self):
+        return self._on
+
+
+class GenericZGPSensor(GenericSensor):
+    def __init__(self, id, raw, request):
+        super().__init__(id, raw, request)
+        self._battery = raw['config'].get('battery')
+        self._lastupdated = raw['state'].get('lastupdated')
+        self._on = raw['config'].get('on')
+
+    @property
+    def battery(self):
+        return self._battery
+
+    @property
+    def lastupdated(self):
+        return self._lastupdated
+
+    @property
+    def on(self):
+        return self._on
+
+
+class GenericZLLSensor(GenericSensor):
+    def __init__(self, id, raw, request):
+        super().__init__(id, raw, request)
+        self._battery = raw['config'].get('battery')
+        self._lastupdated = raw['state'].get('lastupdated')
+        self._on = raw['config'].get('on')
+        self._reachable = raw['config'].get('reachable')
+
+    @property
+    def battery(self):
+        return self._battery
+
+    @property
+    def lastupdated(self):
+        return self._lastupdated
+
+    @property
+    def on(self):
+        return self._on
+
+    @property
+    def reachable(self):
+        return self._reachable
+
+
+class DaylightSensor(GenericSensor):
+    def __init__(self, id, raw, request):
+        super().__init__(id, raw, request)
+        self._daylight = raw['state'].get('daylight')
```

Comment:
We do 'get' on everything. We should only do 'get' if the value might not be there.